packages/babel-preset-mc-app/index.js

```diff
@@ -106,7 +106,8 @@ module.exports = function getBabePresetConfigForMcApp() {
         require('@babel/plugin-transform-runtime').default,
         {
           corejs: 3,
-          helpers: false,
+          // To be able to use `runtime` in Rollup babel plugin
+          helpers: true,
```

Comment:
@tdeekens This does the trick ⬆️

Reply:
Then it was misconfigured before I think. Really odd.

Reply:
No idea why we had it configured that way 🤷‍♂️